README.md

```diff
@@ -48,6 +48,7 @@ conda create -n lmdeploy python=3.10
 conda activate lmdeploy
 git clone https://github.com/InternLM/lmdeploy.git
 cd lmdeploy
+pip install -r requirements.txt
```

Comment:
`pip install  -e .` installs the dependent package automatically. We don't need to do it manually.